addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -651,11 +651,22 @@ function factory(dependencies) {
             const convertedData = this.env.models['mail.message'].convertData(data);
             const messageIds = this.env.models['mail.message'].all().map(message => message.id);
             const partnerRoot = this.env.messaging.partnerRoot;
-            this.env.models['mail.message'].create(Object.assign(convertedData, {
+            const message = this.env.models['mail.message'].create(Object.assign(convertedData, {
                 author: [['link', partnerRoot]],
                 id: (messageIds ? Math.max(...messageIds) : 0) + 0.01,
                 isTransient: true,
             }));
+            for (const channelId of data.channel_ids) {
+                const thread = this.env.models['mail.thread'].findFromIdentifyingData({
+                    id: channelId,
+                    model: 'mail.channel'
+                });
+                thread.threadViews.map(threadView =>
+                    threadView.addComponentHint('current-partner-just-posted-message', {
+                        messageId: message.id
+                    })
+                );
+            }
```

Comment:
Found this quite verbal but didn't find another way to do that.
Not sure either if the component hint I used it the good one even if it does exactly what we want.

What do you guys think @seb-odoo & @alexkuhn ?

Reply:
This should probably be flux specific in the method calling the command, just like the other occurrence of `current-partner-just-posted-message` is in message post itself.

We don't want all tabs or even all viewers to sync with this. Only the specific thread viewer that was used to post the message should have it, as visual feedback the action was correctly done.

All other tabs or viewers can be left to the normal scrolling behavior on new message (scroll to bottom if the list was already scrolled to the bottom, which has to be fixed in task-2336251).

Reply:
I don't know how you can do that as the message is not posted at the moment you write the command. The message is created when you get the notification from the server. Should we create a "placeholder" message ?

Reply:
`_handleNotificationPartnerTransientMessage` where we create the fake id would be a good place to put it then

Reply:
resolving this one then